packages/maumau-server/test/reducer.test.ts

```diff
@@ -23,4 +27,20 @@ describe('reducer', () => {
       expect(newState.stack.includes(cardToPlay)).toBeTruthy();
     });
   });
+
+  describe("when ACCEPT_PENDING_SEVENS action", () => {
+
+    const state = initalizeGame(2);
```

Comment:
```suggestion
    let state = initalizeGame(2);
```

lol ich habs kaputt gemacht.